front/inventory.conf.php

```diff
@@ -40,11 +40,26 @@
 Html::header(__('Inventory'), $_SERVER['PHP_SELF'], "admin", "glpi\inventory\inventory");
 
 $conf = new Conf();
+$glpikey = new GLPIKey();
 
 if (isset($_FILES['inventory_files'])) {
     $conf->displayImportFiles($_FILES);
 } elseif (isset($_POST['update'])) {
     unset($_POST['update']);
+    if (
+        (
+            !$_POST['basic_auth_password'] ||
+            !$_POST['basic_auth_login']
+        ) && $_POST['auth_required'] === Conf::BASIC_AUTH
+    ) {
+        Session::addMessageAfterRedirect(
+            "Basic Authentication is active. The login and/or password fields are missing.",
+            false,
+            ERROR
+        );
+        Html::back();
+    }
+    $_POST['basic_auth_password'] = $glpikey->encrypt($_POST['basic_auth_password']);
```

Comment:
This part should be place in  `src/Inventory/Conf.php` (`saveConf()`)

And return `false` if `login` or `pwd` are empty